components/molecules/FilterCardSelect/filter-card-select.tsx

```diff
@@ -58,6 +60,16 @@ const FilterCardSelect: React.FC<FilterCardSelectProps> = ({
         <SelectValue placeholder="select topic" />
       </SelectTrigger>
       <SelectContent className="bg-white">
+        {customTopic && (
```

Comment:
Just wondering how we guard about putting anything here. If I add `potato`, it still renders in the list.

![CleanShot 2024-02-13 at 08 10 32](https://github.com/open-sauced/app/assets/833231/e1efc289-1738-4132-87a4-1a7197684607)

I know these routes need some fixing, but just wondering if it makes more sense to add it to the list directly in https://github.com/open-sauced/app/blob/beta/lib/utils/getInterestOptions.ts ?

Reply:
Sure, we can add that one to the list, but we can't add all the common ones. That's we why allow custom ones to be done through the URL as a bit of a hidden feature.

Reply:
I guess we can longterm validate it, so we don't have potato. 🥔